### ISSUE TITLE
White screen after launch

### ISSUE THREAD

**Issue by koasmox** [Catfriend1/syncthing-android#692]
I just want to report the issue in case other people experience it, too. I will publish more info when there is need to solve the issue.

### Description of the issue

- Suddenly, after having it not opened for a while, launching the app shows a white screen and nothing else happens.
- Clearing the cache did not have any effect.
- Deleting all app data made the introduction tour show up on launch; after that again a white screen appeared.
- Reinstalling restored full functionality.

### Reproducer

–

### Version Information
-    App Version: 1.8.0.1 (F-Droid)
-    Syncthing Version: v1.8.0-preview.3
-    Android Version: LineageOS 16.0

**Comment by Catfriend1** [comment]
Hi,

Thanks for reaching out. I didn't know this issue yet, sounds interesting what's going on. Could it be a Lineage bug? Does it happen on other phones with different firmware too? What happens if you install Syncthing official?

**Comment by koasmox** [comment]
Thanks for the quick response.

> Could it be a Lineage bug?

I cannot exclude that. However, I do not remember any similar issues with other apps. Another possibility is that the file system was partially damaged.

> Does it happen on other phones with different firmware too?

On my Samsung Tablet it works smoothly.

> What happens if you install Syncthing official?

I did not have it installed before. I installed it now and it seems to work. As I said, reinstalling your fork restored full functionality of the app. So the priority is fairly low.
